### ISSUE TITLE
Feature Request: Restore Demo Content

### ISSUE THREAD

**Issue by Abourass** [zadam/trilium#352]
As I've recommended this app to a few friends, about half of them deleted the demo on opening thinking it would be simple enough to learn with out examples. As the initial learning curve for this notebook can be a tad higher, from being so feature rich, it'd be nice if there was an option to restore the demo content without having to delete the database and setting it up a second time.

**Comment by terafo** [comment]
You can restore it if you import demo.tar that you can download [here](https://github.com/zadam/trilium/raw/master/db/demo.tar). You can also delete trilium-data folder and it restores on its own. In previous version, it was possible to find demo in the app files, but since latest version due to changes in the building process they are now in asar archives and it's easier just to download demo. I think it would be a good idea to move demo and other example files(if there will be any new) to the separate folder.

**Comment by zadam** [comment]
Yep, it is as terafo says. Initial demo document is just standard import done automatically after initialization. Unfortunately it's now inside ASAR file, but it should be possible to put it outside ...

**Comment by rhnvrm** [comment]
I deleted the demo because I thought I would be able to set up the scripts myself. I could not figure that out, maybe adding the actual scripts and their setup to the wiki would help new users like me. One more suggestion would be to keep the demo notes etc separate from the scripts so that users can just delete the journal entries and not the actual scripts.

But, apart from that, now even after importing the demo archive, the buttons on the top right are still missing. I'm not sure what is causing that or if I am missing something

**Comment by freshfriedfish** [comment]
still missing buttons even after importing demo and even reinstalling

**Comment by rubaboo** [comment]
> suggestion would be to keep the demo notes etc separate from the scripts so that users can just delete the journal entries and not the actual scripts.

This. This is what I'm talking about in #1036. The demo says you can delete it when you are done, but it's hard for a new user to realize that a lot of functionalty that he/she is seeing is actually part of the Demo. Like, yeah, go ahead and delete the Journal - oops, there is no Today button in the sidebar anymore :(.

The Demo needs to be separated into 2 parts - 

1. A disposable Demo tree.
2. "Add-ons" tree containing all the functional bits.
